src/main/webapp/js/countryCodes.js

```diff
@@ -0,0 +1,283 @@
+    'Micronesia, Federated States Of': 'FSM',
+    'Moldova': 'MDA',
+    'Monaco': 'MCO',
+    'Mongolia': 'MNG',
+    'Montenegro': 'MNE',
+    'Montserrat': 'MSR',
+    'Morocco': 'MAR',
+    'Mozambique': 'MOZ',
+    'Myanmar': 'MMR',
+    'Namibia': 'NAM',
+    'Nauru': 'NRU',
+    'Nepal': 'NPL',
+    'Netherland': 'NLD',
+    'Netherlands': 'NLD',
+    'New Caledonia': 'NCL',
+    'New Zealand': 'NZL',
+    'Nicaragua': 'NIC',
+    'Niger': 'NER',
+    'Nigeria': 'NGA',
+    'Niue': 'NIU',
+    'Norfolk Island': 'NFK',
+    'Northern Mariana Islands': 'MNP',
+    'Norway': 'NOR',
+    'Oman': 'OMN',
+    'Pakistan': 'PAK',
+    'Palau': 'PLW',
+    'Palestinian Territory, Occupied': 'PSE',
+    'Panama': 'PAN',
+    'Papua New Guinea': 'PNG',
+    'Paraguay': 'PRY',
+    'Peru': 'PER',
+    'Philippines': 'PHL',
+    'Pitcairn': 'PCN',
+    'Poland': 'POL',
+    'Portugal': 'PRT',
+    'Puerto Rico': 'PRI',
+    'Qatar': 'QAT',
+    'Reunion': 'REU',
+    'Romania': 'ROU',
+    'Russia': 'RUS',
+    'Russian Federation': 'RUS',
+    'Rwanda': 'RWA',
+    'Saint Barthelemy': 'BLM',
+    'Saint Helena': 'SHN',
+    'Saint Kitts And Nevis': 'KNA',
+    'Saint Lucia': 'LCA',
+    'Saint Martin': 'MAF',
+    'Saint Pierre And Miquelon': 'SPM',
+    'Saint Vincent And Grenadines': 'VCT',
+    'Samoa': 'WSM',
+    'San Marino': 'SMR',
+    'Sao Tome And Principe': 'STP',
+    'Saudi Arabia': 'SAU',
+    'Senegal': 'SEN',
+    'Serbia': 'SRB',
+    'Seychelles': 'SYC',
+    'Sierra Leone': 'SLE',
+    'Singapore': 'SGP',
+    'Slovakia': 'SVK',
+    'Slovenia': 'SVN',
+    'Solomon Islands': 'SLB',
+    'Somalia': 'SOM',
+    'South Africa': 'ZAF',
+    'South Georgia And Sandwich Isl.': 'SGS',
+    'Spain': 'ESP',
+    'Sri Lanka': 'LKA',
+    'Sudan': 'SDN',
+    'Suriname': 'SUR',
+    'Svalbard And Jan Mayen': 'SJM',
+    'Swaziland': 'SWZ',
+    'Sweden': 'SWE',
+    'Switzerland': 'CHE',
+    'Syrian Arab Republic': 'SYR',
+    'Taiwan': 'TWN',
+    'Tajikistan': 'TJK',
+    'Tanzania': 'TZA',
+    'Thailand': 'THA',
+    'Timor-Leste': 'TLS',
+    'Togo': 'TGO',
+    'Tokelau': 'TKL',
+    'Tonga': 'TON',
+    'Trinidad And Tobago': 'TTO',
+    'Tunisia': 'TUN',
+    'Turkey': 'TUR',
+    'Turkmenistan': 'TKM',
+    'Turks And Caicos Islands': 'TCA',
+    'Tuvalu': 'TUV',
+    'Uganda': 'UGA',
+    'Ukraine': 'UKR',
+    'UAE': 'ARE',
+    'United Arab Emirates': 'ARE',
+    'United Kingdom': 'GBR',
+    'UK': 'GBR',
+    'US': 'USA',
+    'United States': 'USA',
+    'United States Outlying Islands': 'UMI',
+    'Uruguay': 'URY',
+    'Uzbekistan': 'UZB',
+    'Vanuatu': 'VUT',
+    'Venezuela': 'VEN',
+    'Vietnam': 'VNM',
+    'Viet Nam': 'VNM',
+    'Virgin Islands, British': 'VGB',
+    'Virgin Islands, U.S.': 'VIR',
+    'Wallis And Futuna': 'WLF',
+    'Western Sahara': 'ESH',
+    'Yemen': 'YEM',
+    'Zambia': 'ZMB',
+    'Zimbabwe': 'ZWE'
+};
+
+// Reverse mapping from alpha 3 code to country name
+var codeToCountry = {};
```

Comment:
Actually, why do you need a key-value mapping for this? Won't an array suffice, because the usage later on is just checking the presence of key (equivalent to presence of element in array)?